domain/tutoringModel/utils.py

```diff
@@ -113,3 +113,33 @@ def random_generator(num, size, type_):
         return rng.choice([True, False], size)
     else:
         raise err.WrongParameterValueError()
+
+
+# Check if learning style is correctly formatted and
+# has the right dimensions
+def check_learning_style(input_learning_style):
```

Comment:
more pythonic if returns true when successful. Otherwise might rename to assert_learning_style so it is immanent that it will only catch exceptions.

Reply:
Yep, True sounds better here